### PR TITLE
Feature - CreateWallet

### PR DESCRIPTION
**Pull request by dannypaz**
This PR is based off of https://github.com/sparkswap/lnd-engine/pull/106. DO NOT MERGE before changing base branch back to master

This PR adds a `create-wallet` command to the LND engine, to allow the user to initialize an LND wallet for first use.

Other changes:
1. added `initWallet` method for lnd-actions

### CODE REVIEW

**Comment by dannypaz** [comment]
[Add `createWallet` to lnd-engine](https://trello.com/c/Z38wXYZf/858-add-createwallet-to-lnd-engine)

**Comment by treygriffith** [comment]
@dannypaz can you change the PR base to one of the other branches for comparison purposes?

File: src/lnd-actions/gen-seed.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { deadline } = require('../grpc-utils')
```
Comment (dannypaz) [inline src/lnd-actions/gen-seed.js]: This file is actually added in #106, not sure why it is still showing in the diff

File: src/engine-actions/is-engine-unlocked.spec.js [hunk 2]
```diff
@@ -9,10 +9,7 @@ describe('isEngineUnlocked', () => {
 
   beforeEach(() => {
     engine = {
-      walletUnlocker: sinon.stub(),
-      logger: {
-        error: sinon.stub()
-      }
+      walletUnlocker: sinon.stub()
```
Comment (dannypaz) [inline src/engine-actions/is-engine-unlocked.spec.js]: This change was listed in the merge. There is no usage of `logger` in the spec

**Pull request merged**